app/views/provider_interface/application_choices/show.html.erb

```diff
@@ -52,7 +52,9 @@
     <%= render ProviderInterface::TrainingWithDisabilityComponent.new(application_form: @application_choice.application_form) %>
 
     <% if FeatureFlag.active?('provider_view_safeguarding') %>
-      <%= render ProviderInterface::SafeguardingDeclarationComponent.new(application_form: @application_choice.application_form)%>
+      <% if current_provider_user.can_view_safeguarding_information_for?(@application_choice.provider) %>
```

Comment:
I wonder if this should be:

```suggestion
      <% if current_provider_user.can_view_safeguarding_information_for?(@application_choice.offered_option.provider) %>
```

offered_option will return the original course option unless an alternative one has been offered by the provider. The alternative one may be belong to a different provider that the user is associated with.

Reply:
This is a good question! I think we should tackle it separately to this PR, though?